File: bookStore v.0.02/Models/Book.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace bookStore_v._0._02.Models
{
    class Book
    {
        public int BookID { get; set; }
        public string ISBN { get; set; }
        public string Title { get; set; }
        public string Subtitle { get; set; }
        public string Author { get; set; }
        public string Publisher { get; set; }
        public int Pages { get; set; }
        public string Description { get; set; }
        public decimal Value { get; set; }
        public int AmountInStock { get; set; }
    }
}

File: BookStore v.0.3/BookStore v.0.3/ClientApp/src/app/models/book.ts
export class Book {
  constructor(
    public id?: number,
    public isbn?: string,
    public title?: string,
    public subtitle?: string,
    public author?: string,
    public publisher?: string,
    public pages?: number,
    public value?: number,
    public amountInStock?: number,
    public description?: string) { }
}
